src/components/FolderPage/CardListItem/CardListItem.js

```diff
@@ -3,31 +3,110 @@ import { getTimeAgo, formatDate } from "util/time";
 import noImg from "assets/noImg.png";
 import starImg from "assets/Star 1.svg";
 import kebabImg from "assets/kebab.svg";
+import { useEffect, useRef, useState } from "react";
+
+function CardListItem({ link, setModal, setLink }) {
+  const [btnclicked, setBtnClicked] = useState(false);
```

Comment:
state명도 setter함수처럼 카멜케이스로 맞추면 좋을 것 같아요!
```suggestion
  const [btnClicked, setBtnClicked] = useState(false);
```